plugins/templates/dashboard.html

```diff
@@ -3,102 +3,168 @@
-          <th class='column-header'>Most Recent Run Status</th>
-          <th class='column-header'>Most Recent Run Date</th>
-          <th class='column-header'>Next Scheduled</th>
-        </tr>
-      </thead>
-
-      {% for dag in data %}
-        <tr>
-          <td>
-            <strong>{{ dag.name }}</strong>
-            <i class="fa fa-info-circle" data-toggle="tooltip" data-placement="top" title="{{ dag.description }}"></i>
-          </td>
-          {% if dag.run_state == 'failed' %}
-            <td class="text-center"><span style="color: red"><i class="fa fa-times-circle"></i><br />Failed</span></td>
-          {% elif dag.run_state == 'success' %}
-            <td class="text-center"><span style="color: green"><i class="fa fa-check-circle"></i><br />Succeeded</span></td>
-          {% elif dag.run_state == 'running' %}
-            <td class="text-center"><i class="fa fa-spinner fa-spin"></i><br />Running</td>
-          {% else %}
-            <td class="text-center">{{ dag.run_state or '' }}</td>
-          {% endif %}
 
-          {% if dag.run_date != None %}
-            <td>{{ dag.run_date.pst_time }} PST<br /> {{ dag.run_date.cst_time }} CST</td>
+          <strong>Last Successful Event Run</strong><br />
+          {% if event_last_run %}
+            <table class="table table-sm">
+            <thead>
+              <th>DAG</th>
+              <th>Time</th>
+            </thead>
+            <tr>
+              <td>
+                {{ event_last_run.dag_id }}
+              </td>
+              <td>
+                {{ event_last_run_time.pst_time.strftime(datetime_format) }} PST<br />
+                {{ event_last_run_time.cst_time.strftime(datetime_format) }} CST
+              </td>
+            </tr>
+          </table>
           {% else %}
-            <td></td>
+            No Event Scrapes Completed
           {% endif %}
-          {% if dag.next_scheduled != None %}
-            <td>{{ dag.next_scheduled.pst_time }} PST<br /> {{ dag.next_scheduled.cst_time }} CST</td>
+
+          <strong>Upcoming Event Run</strong><br />
+          {% if event_next_run %}
+            <table class="table table-sm">
+              <thead>
+                <th>DAG</th>
+                <th>Time</th>
+              </thead>
+              <tr>
+                <td>
+                  {{ event_next_run.name }}
+                </td>
+                <td>
+                  {{ event_next_run.next_scheduled.pst_time.strftime(datetime_format) }} PST<br />
+                  {{ event_next_run.next_scheduled.cst_time.strftime(datetime_format) }} CST
+                </td>
+              </tr>
+            </table>
           {% else %}
-            <td></td>
+            No Event Scrapes Scheduled
           {% endif %}
-        </tr>
-      {% endfor %}
-    </table>
-  {% endblock %}
+        </div>
+      </div>
+
+    </div><!-- end .row -->
+
+    <hr class="m-3" />
+
+    {% block model_list_table %}
+    <div class="row">
```

Comment:
The only change to this section is that I wrapped it in row/col divs, i.e., everything has been indented two levels.